query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java

```diff
@@ -209,48 +207,10 @@ public static Expression createTimeColumnGroupByExpression(String timeColumn, St
         .build();
   }
 
-  public static QueryRequest getQueryRequestWithTimeFilter1(Duration timeRangeDuration) {
-    long startTimeInMillis = TimeUnit.MILLISECONDS.convert(Duration.ofHours(1));
+  public static QueryRequest getQueryRequestWithTimeFilter(Filter filter) {
```

Comment:
nit: there's nothing "time" about this filter
```suggestion
  public static QueryRequest getQueryRequestWithFilter(Filter filter) {
```